greynoise/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
NAK - I appreciate the liberal license, but all new packs in Exchange MUST be licensed under the Apache 2 License. That way all users who can install StackStorm itself know that they can also install any pack on the StackStorm Exchange (although I realize that the MIT license is somewhat less restrictive than the Apache license, consistency make everybody's lives easier in this regard).

There are two packs on Exchange that have been grandfathered into this policy, which I think is probably where you got the impression that MIT was okay. Both of those packs have PRs changing their licensing, but both are held up by contributors who haven't accepted the PR changes (and unfortunately may never do so).

Reply:
Updated.  Please validate I used the correct one now.